File: src/main/java/com/eason/Hello.java
package com.eason;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello World");
//            new Person().hello();
//            Person p = new Person();  //Person()系統預設無傳參數建構子
            Person p = new Person("Eason",1.7, 85.5);
            p.hello();
            p.height = 1.7;
            p.weight = 85.5;
            p.name = "Eason";
            System.out.println(p.name);
            System.out.println(p.bmi());

            /*int age = 19;   //short long
            Integer age2 = 19;
            char c = '我';
            Character c2 = c;
            byte b = 120;
            float weight = 66.5f;
            double height = 1.7;
            boolean adult = true;
            boolean enroll = false;
            String name = "Eason";*/
    }
}

File: src/main/kotlin/com/eason/Hello.kt
package com.eason

fun main(args: Array<String>) {
//    println("Hello Kotlin")
//        Human().hello()
        val h = Human(weight = 66.5f,height = 1.7f)
        println(h.bmi())
        h.hello()
        val score = 88
        println(score >80)
        val c:Char = 'A'
        println(c.toInt()>60)
        /*var age = 19    //val 不可再改變
        age = 20
        var weight = 66.5
        val String = "Eason"
        val name : String
        name = "Eason2"*/
}

class Human(var name: String = "",var weight:Float, var height:Float){
    init {
        println("test $weight")
    }

    fun bmi(): Float {
        val bmi = weight / (height*height)
        return bmi
    }
    fun hello(){
        println("Hello Kotlin")
    }
}